core/base/test/CMakeLists.txt

```diff
@@ -9,14 +9,14 @@
 # onepcm targets to be built.
 
 if(NOT MSVC)
-  set(dllib dl)
+  set(extralibs Cling dl)
```

Comment:
should probably add a comment that explains why we need this, for future ROOT devs

Reply:
> should probably add a comment that explains why we need this, for future ROOT devs

The comment is already [there](https://github.com/root-project/root/blob/master/core/base/test/CMakeLists.txt#L7-L9)

Reply:
that comment doesn't explain why Windows needs special treatment, it just says why we need to link `libCilng` besides `libCore`